ui/angular.json

```diff
@@ -104,7 +104,11 @@
         "serve": {
           "builder": "@angular-devkit/build-angular:dev-server",
           "options": {
-            "browserTarget": "orb-ui:build"
+            "browserTarget": "orb-ui:build",
+            "ssl": true,
+            "sslKey": "scripts/key.pem",
+            "sslCert": "scripts/cert.pem",
+            "disableHostCheck": true
```

Comment:
@iatdaitan I think these changes shouldn't be available, and I have created a separate gist for it [here](https://gist.github.com/gpazuch/dd853646b4554fb2d8c9a6fa30ce4479).

In order to use it in local environment for development/testing purposes, one must create a certificate - in this case, placed inside `scripts` folder

```openssl req -newkey rsa:2048 -new -nodes -x509 -days 3650 -keyout key.pem -out cert.pem```

as well as have the IC keys available in the host env.

```
export PS_GROUP_KEY=my-group-key
export PS_SID=my-site-id
```

before running `yarn start` to serve the application.

@manrodrigues with this you should be able to test ironclad locally.

Reply:
Thank you.